java/src/org/openqa/selenium/json/JsonOutput.java

```diff
@@ -110,159 +112,169 @@ public class JsonOutput implements Closeable {
+        ((Map<?, ?>) obj)
+          .forEach(
+            (key, value) -> {
+              if (value instanceof Optional && !((Optional) value).isPresent()) {
+                return;
+              }
+              name(String.valueOf(key)).write0(value, maxDepth, depthRemaining - 1);
+            });
+        endObject();
+      });
     builder.put(
-        Class::isArray,
-        (obj, maxDepth, depthRemaining) -> {
-          if (depthRemaining < 1) {
-            throw new JsonException(
-                "Reached the maximum depth of " + maxDepth + " while writing JSON");
-          }
-          beginArray();
-          Stream.of((Object[]) obj)
-              .filter(o -> (!(o instanceof Optional) || ((Optional<?>) o).isPresent()))
-              .forEach(o -> write0(o, maxDepth, depthRemaining - 1));
-          endArray();
-        });
+      Class::isArray,
+      (obj, maxDepth, depthRemaining) -> {
+        if (depthRemaining < 1) {
+          throw new JsonException(
+            "Reached the maximum depth of " + maxDepth + " while writing JSON");
+        }
+        beginArray();
+        Stream.of((Object[]) obj)
+          .filter(o -> (!(o instanceof Optional) || ((Optional<?>) o).isPresent()))
+          .forEach(o -> write0(o, maxDepth, depthRemaining - 1));
+        endArray();
+      });
 
     builder.put(
-        Optional.class::isAssignableFrom,
-        (obj, maxDepth, depthRemaining) -> {
-          Optional<?> optional = (Optional<?>) obj;
-          if (!optional.isPresent()) {
-            append("null");
-            return;
-          }
+      Optional.class::isAssignableFrom,
+      (obj, maxDepth, depthRemaining) -> {
+        Optional<?> optional = (Optional<?>) obj;
+        if (!optional.isPresent()) {
+          append("null");
+          return;
+        }
 
-          write0(optional.get(), maxDepth, depthRemaining);
-        });
+        write0(optional.get(), maxDepth, depthRemaining);
+      });
 
     // Finally, attempt to convert as an object
     builder.put(
-        cls -> true,
-        (obj, maxDepth, depthRemaining) -> {
-          if (depthRemaining < 1) {
-            throw new JsonException(
-                "Reached the maximum depth of " + maxDepth + " while writing JSON");
-          }
-          mapObject(obj, maxDepth, depthRemaining - 1);
-        });
+      cls -> true,
+      (obj, maxDepth, depthRemaining) -> {
+        if (depthRemaining < 1) {
+          throw new JsonException(
+            "Reached the maximum depth of " + maxDepth + " while writing JSON");
+        }
+        mapObject(obj, maxDepth, depthRemaining - 1);
+      });
 
     this.converters = Collections.unmodifiableMap(builder);
   }
 
   /**
-   * @param enablePrettyPrinting
-   * @return
+   * Specify whether the serialized JSON object should br formatted with line breaks and indention ("pretty printed").
+   * @param enablePrettyPrinting {@code false} for compact format; {@code true} for "pretty printing"
```

Comment:
The formatting here looks really odd. How come the `default` is so far indented?

Reply:
This formatting was applied by IntelliJ when I inserted a line break. The `format` script has cleaned everything up now.